File: uri_exercicios_em_Cpp/beginner/age_in_days.cpp
#include <iostream>
using namespace std;

int age_in_days(){

    int dias;

    cin >> dias;

    cout << dias/365 << " ano(s)\n"<< dias%365/30 << " mes(es)\n"<< dias%365%30 << " dia(s)"<< endl;
    return 0;
}


int main(){

    age_in_days();

    return 0;
}

File: beecrowd_exercicios_em_python/begginner/age_in_days.py
def age_in_days(x):
    return [x // 365, x % 365 // 30, x % 365 % 30]


dias = int(input())

print(
    "{} ano(s)\n{} mes(es)\n{} dia(s)".format(
        age_in_days(dias)[0],
        age_in_days(dias)[1],
        age_in_days(dias)[2])
)
